eventuals/event-loop.h

```diff
@@ -841,9 +851,13 @@ inline auto& Clock() {
 ////////////////////////////////////////////////////////////////////////
 
 inline std::chrono::nanoseconds EventLoop::Clock::Now() {
+  CHECK(!loop_.running_);
+
   if (Paused()) { // TODO(benh): add 'unlikely()'.
     return *paused_ + advanced_;
   } else {
+    // uv_now() doesn't update time by itself.
+    uv_update_time(loop_);
```

Comment:
Hmm, I'm still not sure based on this comment why we need to update the time. I believe you that you ran into an issue, I just don't know what it is and need you to elaborate this comment because it sounds tricky!

Reply:
Added a comment!

Reply:
Seems like we're still missing a comment that explains why we ned to do this regardless of the fact that `uv_now()` doesn't update time by itself?